scripts/importer.py

```
@@ -0,0 +1,359 @@
+    except ImportError:
+        print("Install driver: pip install psycopg2-binary")
+        sys.exit(1)
+
+    staging = _load_staging()
+    approved = [s for s in staging["shops"] if s["approved"]]
+
+    if not approved:
+        print('No approved shops. Run "approve" first or edit staging.json.')
+        sys.exit(1)
+
+    print(f"Connecting to DB...")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)